src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/EntityModal/addEntityModal.html.twig

```diff
@@ -30,7 +36,18 @@
                 </ul>
             </fieldset>
             <fieldset class="add-entity-fieldset add-entity-template" aria-describedby="fieldset3">
-                <h5 class="add-entity-question" id="fieldset3">{{ 'entity.add.template.title'|trans }}</h5>
+                {% set templateTitle = 'entity.add.template.title'|trans %}
+                {% set allowedTags = 'allowed.html.tags'|trans %}
+                <input type="checkbox" name="templateTooltip" id="templateTooltip" hidden class="tooltip tooltipCheckbox" />
+                <h5 class="add-entity-question" id="fieldset3">
+                    <span class="title">{{ templateTitle }}</span>
+                    <label for="templateTooltip" class="tooltip tooltipLabel">
+                        {{ 'entity.add.template.title.tooltip.srText'|trans }}
+                    </label>
+                </h5>
+                <div class="tooltip tooltipContent">
+                    {{ 'entity.add.template.title.tooltip.text'|striptags('<br><wbr><p><a><ul><ol><li><dl><dd><dt><blockquote><cite><strong><em><div><span><address><time>')|trans|raw }}
```

Comment:
Thank you for adding the filter on the allowed tags. I recon this is a sane set of allowed elements!